ui/components/ui/ui-components.scss

```
@@ -61,4 +61,5 @@
 @import 'deprecated-test-networks/index.scss';
 @import 'nft-info/index.scss';
 @import 'nft-collection-image/index';
-
+@import '../institutional/custody-labels/index';
+@import '../institutional//note-to-trader/index';
```

```suggestion
@import '../institutional/note-to-trader/index';
```
This seems like a typo?